configure

```diff
@@ -204,7 +204,7 @@ fi
 
 # If the user asked for classic netcdf, acquiesce to the request.
 
-if [ "`uname`" = "Linux" ] ; then
+if [ "`uname`" = "Linux" -o "`uname -o`" = "Cygwin" ] ; then
```

Comment:
Is uname -o correct?

Reply:
`uname` gives result `CYGWIN_NT-10.0`.  I could change that, but I'm not sure when the "10.0" part changes.

Reply:
The -o option trims the "10.0" and changes the string from "CYGWIN_NT" to "Cygwin"?